### PR TITLE
[FIX] storybook errors

### PR DESCRIPTION
**Pull request by ishak-lm**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   "scripts": {
     "clean": "rimraf coverage build",
     "prebuild": "yarn clean",
-    "storybook": "start-storybook --config-dir src/storybook/config",
+    "storybook": "start-storybook --config-dir src/storybook/config -p 3456",
```
Comment (fubhy) [inline package.json]: Why this change?

File: src/components/Contexts/Color/Color.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const ColorProvider: React.FC<ColorProviderProps> = props => {
 
   useLayoutEffect(() => {
     setBase(color || randomColor());
-  }, [location.pathname]);
+  }, [location.pathname, color]);
```
Comment (fubhy) [inline src/components/Contexts/Color/Color.tsx]: This is not correct. The effect was used here without listening to `color` changes so that it would only update on route change.

File: package.json (hunk 1 above)
Comment (ishak-lm) [reply package.json]: Because the PORT change all the time it's more easier to have the same one

Comment (fubhy) [reply package.json]: Ok, makes sense!
